package.json

```diff
@@ -5,24 +5,25 @@
   "homepage": "/",
   "license": "GPL-3.0-or-later",
   "engines": {
-    "node": "^16.18.1"
+    "node": "~18.15.0"
   },
   "scripts": {
+    "preview": "vite preview",
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "build": "yarn i18n && env-cmd -f .env react-app-rewired build",
-    "build-dev": "yarn i18n && env-cmd -f .env.dev react-app-rewired build",
-    "build-prod": "yarn i18n && env-cmd -f .env.production react-app-rewired build",
-    "build-stg": "yarn i18n && env-cmd -f .env.stg react-app-rewired build",
+    "build": "yarn i18n && tsc && env-cmd -f .env vite build",
```

Comment:
The order of tsc and vite build is different from the other commands

Reply:
this one is correct

Reply:
will fix those other